### ISSUE TITLE
Patio Light auto turned on after manually turned off at night

### ISSUE THREAD

**Issue by frederickcwong** [frederickcwong/jarvis#23]
Question [posted to HA's forum](https://community.home-assistant.io/t/puzzled-by-sunset-sunrise-lights-on-automation/470356):

![Image](https://user-images.githubusercontent.com/77422662/194172774-2cd122b6-3e8f-44b4-8448-4d7adb7eed43.png)

**Comment by frederickcwong** [comment]
![Image](https://user-images.githubusercontent.com/77422662/194198900-2811a29c-37c2-461b-859a-3401ab0fd048.png)

**Comment by frederickcwong** [comment]
Looks like the "state" restriction works. However, the light still turned on after I turned it off last night. Looking at the logbook, the "no motion after 5 mins" was triggered at that time. The issue is one of the action options was only checking if the sun is `below_horizon` (for the sunset trigger). This condition was satisfied and therefore executed, turning the light on at 5% brightness.

The solution I found is to make sure the action will be executed only when it is triggered by the state-changed event. This is done by adding a custom trigger id in the trigger section of the automation and then using the ID as a condition for the action to check. These are added to the automation and wait for the confirmation later tonight.
